docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -29,6 +25,22 @@ and then read metadata from each data file.
 Since Iceberg stores the paths to data files in the metadata files, it
 only consults the underlying file system for files that must be read.
 
+Requirements
+------------
+
+To connect to Iceberg, you need:
+
+* Iceberg data files stored in either Parquet or ORC format, as determined by
+  the ``format`` property in the table definition. The table ``format`` defaults
+  to ``ORC``.
+* Iceberg table format compatible with Apache Iceberg version 0.11.0.
+* Network access from the Trino coordinator and workers to the distributed
+  object storage.
+* Hive metastore service (HMS), or a compatible implementation of the Hive
+  metastore.
+* Network access from the Trino coordinator and workers to the HMS. Hive
```

Comment:
Iceberg actually only needs coordinator access to HMS. (unlike the Hive connector, which needs HMS access on workers when doing writes to partitioned tables)